control-plane/api/v1alpha1/exportedservices_types.go

```diff
@@ -29,6 +29,7 @@ func init() {
 // +kubebuilder:printcolumn:name="Synced",type="string",JSONPath=".status.conditions[?(@.type==\"Synced\")].status",description="The sync status of the resource with Consul"
 // +kubebuilder:printcolumn:name="Last Synced",type="date",JSONPath=".status.lastSyncedTime",description="The last successful synced time of the resource with Consul"
 // +kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp",description="The age of the resource"
+// +kubebuilder:resource:shortName="exported-services"
```

Comment:
it's funny that it is the "short" name even though it is a character longer.

Reply:
😂 I know I'm stretching the recommended usage... but they don't have an `aliases` field!